### ISSUE TITLE
ClassPath doesn't refresh

### ISSUE THREAD

**Issue by martinambrus** [codenvy/codenvy#2294]
On CodeEnvy 5.13.0 (Java project) when I add a local JAR file into the classpath OR when I execute a build command (which copies a JAR file into a location provided in the .che/classpath file), the actual classpath does not seem to register either of these actions and keeps showing "the import ... cannot be resolved" errors in the editor.

![image](https://user-images.githubusercontent.com/7406402/27734153-da4e8634-5d99-11e7-88f5-19fbef89d1f2.png)

![image](https://user-images.githubusercontent.com/7406402/27734164-e4e6cd72-5d99-11e7-8755-89ea5542e54a.png)

**Reproduction Steps:**
1. create new Java project
2. create new Java file with an unresolvable import
3. copy the required JAR file into a reachable local folder
4. adjust classpath via Project -> Configure classpath
5. observe import errors that remain in editor

**Codenvy version:**
5.13.0

**OS and version:**    
Ubuntu 16.04 64-bit, latest Chrome (Version 59.0.3071.115 (Official Build) (64-bit))

**Docker version:**
N/A (local to the Che on Codenvy)

**Codenvy cli.log output:**
N/A

**Comment by eivantsov** [comment]
@martinambrus I have tried to reproduce this bug but failed. What I did is:

1. Created a simple Java project
2. Uploaded selenium-server jar to `bin`
3. Added it to classpath

![image](https://user-images.githubusercontent.com/5337267/28256078-66e52a00-6ac7-11e7-8f98-22876701585d.png)

I think the problem is that you expect errors to disappear right after and external lib has been added to classpath. If you reopen the file, the errors are gone.

I\ve registered an issue in Che project - https://github.com/eclipse/che/issues/5654
